commercetools-internal-processors/src/main/java/io/sphere/sdk/annotations/processors/models/PropertyGenModel.java

```diff
@@ -79,6 +85,15 @@ public TypeName getType() {
     }
 
     /**
+     * Returns the getter method name for this property.
+     *
+     * @return the getter method name
+     */
+    public String getMethodName() {
```

Comment:
i like that you added the getter method name 👍  it's used everywhere so it's good to factorize the code here